JsonPath/JsonSerializerOptionsExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Json.Path;
+
+public static class JsonSerializerOptionsExtensions
+{
+	public static JsonSerializerOptions WithJsonPath(this JsonSerializerOptions options)
```

Comment:
I don't think this is necessary, since JsonPath library doesn't have any types it needs for its converters to succeed. The pattern is needed if we have TypeInfos we need to dynamically look up in the TypeInfo dictionary to serialize types in this library, but there aren't any.

Reply:
I may not be, but a user might.  Would this not help them?

Reply:
If they to serialize a JsonPath they should do [JsonSerializable(JsonPath)] on their own JsonSerializerContext. The pattern is: if you want to serialize those types, you should generate a JsonTypeInfo for them in your own assembly and be self-complete.